ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSmain.cpp

```diff
@@ -63,6 +63,30 @@ HWND get_window_handle() {
 
 }  // namespace enigma
 
+namespace enigma_user {
+  double set_working_directory(string dname)
+  {
+    replace(dname.begin(), dname.end(), '/', '\\');
+
+    if (!dname.empty())
```

Comment:
Try to be consistent with whether your braces are on the same line as the condition/loop. As an initiative, we're trying to put braces on the same line.